src/views/job-role-edit.njk

```diff
@@ -190,6 +190,7 @@
 </div>
 {% endif %}
 
+<script>
 document.addEventListener('DOMContentLoaded', function() {
     const form = document.getElementById('editJobRoleForm');
     
```

Comment:
This is quite a lot of JS in a view, I think it'd be good to refactor this into a `public/js` folder and use that in the view